service/step_handlers_test.go

```diff
@@ -791,7 +802,7 @@ func handleFileSystems(w http.ResponseWriter, r *http.Request) {
 			replacementMap["__ID__"] = id
 			replacementMap["__NAME__"] = name
 			replacementMap["__SIZE_IN_Total__"] = fileSystemIDToSizeTotal[id]
-			replacementMap["__PARENT_ID__"] = fileSystemIDParentID[id]
+			//replacementMap["__IS_QUOTA_ENABLED__"] = fileSystemIDToQuotaEnabled[id]
```

Comment:
why are you replacing the parent id to quota?

Reply:
updated